app/components/common/DetailSlice.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { View, StyleSheet } from 'react-native'
+import RNText from './RNText'
+import colors from '../../config/colors'
+
+const DetailSlice = ({ title, content }) => {
+  return (
+    <View style={styles.container}>
+      <RNText style={styles.title}>{title}</RNText>
+      <RNText style={styles.content}>{content}</RNText>
+    </View>
+  )
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    flexDirection: 'column',
+    marginLeft: 20,
+    marginRight: 20,
```

Comment:
Just a little thing but you can swap out the left and right margin property and use marginHorizontal for both. Saves you a little bit of work and makes sure both values are always the same when they should be.